exception-handling
nextHandlerContext

	^ self outerContext findNextHandlerContext